src/components/views/settings/tabs/user/SecurityUserSettingsTab.tsx

```diff
@@ -357,7 +357,7 @@ export default class SecurityUserSettingsTab extends React.Component<IProps, ISt
                 { warning }
                 <div className="mx_SettingsTab_heading">{ _t("Where you're signed in") }</div>
                 <div className="mx_SettingsTab_section">
-                    <span>
+                    <span className="mx_SettingsTab_subsectionText">
```

Comment:
I think this class is applied to divs elsewhere

Reply:
From a quick look, it seems to be applied in most places. The only place where I see it not being applied is `User settings` -> `Preferences` -> `Communities` though I think that was intentional?